sites/all/modules/_custom/forms/static/nc_common_functions.js

```diff
@@ -0,0 +1,33 @@
+jQuery(document).ready(function($) {
+	//attach functions to global objects as this serves other files and is in ready callback
+
+	//checks if new folder name/model name is empty
+	window.isNameEmpty = function(name){
+		return ( (name.length > 0) ? false: true);
+	}
```

Comment:
any particular reason that these functions are in window scope?

Reply:
I came across two options here
to make a function global inside document.ready 
either I attach them to window object or 
declare the functions outside and define the functions inside ( var func; $().ready({ func = { $ ... } }). Any better ideas ?

Reply:
It depends on where they are used ? I don't see that so I don't really understand what you are trying to do with them.